no of sub arrays

```diff
@@ -0,0 +1,47 @@
+// Time Complexity : o(n);
```

Comment:
For any contiguous sub-array problem you have to explore three concepts
1. Running sum
2. Two pointers
3. Sliding window
when It comes to sum of subarray running sum is the best option.